english/math/math.md

```diff
@@ -1,5 +1,11 @@
-I'm thinking of an order of infinity between 12 and infinity.
+Hmm.. I'm thinking of an order of infinity between 12 and infinity.
 
 What order of infinity am I thinking of?
 
 -Georg Cantor
+
+Do you: 
+
+[Call your mother.](../tell-mother/tell-mother.md)
+
+[Go back.](../)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.